modules/docs/arrow-docs/docs/docs/arrow-core/patterns/errorhandling/README.md

```diff
@@ -298,59 +298,59 @@ Try.monadError()
 ```kotlin:ank
 import arrow.core.extensions.either.monadError.*
 
-Either.monadError<NukeException>()
+Either.monadError<CookingException>()
 ```
 
 Let's now rewrite our program as a polymorphic function that will work over any datatype for which a `MonadError` instance exists.
 Polymorphic code in Arrow is based on emulated `Higher Kinds`, as described in [Lightweight higher-kinded polymorphism](https://www.cl.cam.ac.uk/~jdy22/papers/lightweight-higher-kinded-polymorphism.pdf) and applied to Kotlin, a lang which does not yet support Higher Kinded Types.
 
 ```kotlin
-fun <f> arm(ME: MonadError<F, NukeException>): Kind<F, Nuke> = ME.just(Nuke)
-fun <f> aim(ME: MonadError<F, NukeException>): Kind<F, Target> = ME.just(Target)
-fun <f> launch(target: Target, nuke: Nuke, ME: MonadError<F, NukeException>):
-  Kind<F, Impacted> = ME.raiseError(MissedByMeters(5))
+fun <F> takeFoodFromRefrigerator(ME: MonadError<F, CookingException>): Kind<F, Lettuce> = ME.just(Lettuce)
+fun <F> getKnife(ME: MonadError<F, CookingException>): Kind<F, Knife> = ME.just(Knife)
```

Comment:
Someone pointed out that this version of `aim`/`getKnife` takes `ME` as parameter instead of an extfun, when in the snippet below for prepare it's used in that position.

Reply:
Oh I saw you mentioned it. Yeah make it an extfun.